backend/apis/clients/list-clients.api.js

```diff
@@ -1,23 +1,42 @@
-const { app, databaseError, pool, invalidRequiest } = require("../../server");
+const { app, databaseError, pool, invalidRequest } = require("../../server");
 const mysql = require("mysql");
 
 app.get("/api/clients", (req, res, next) => {
   pool.getConnection((err, connection) => {
     if (err) {
       return databaseError(req, res, err);
     }
-    var queryString = "SELECT ? AS result";
 
-    const query = mysql.format(queryString, [req.query.foo]);
+    var queryString =
```

Comment:
nitpick - this can be a `const` instead of `var`. The rule of thumb for modern javascript is to start all variables as `const` and then change them to `let` if you need to modify its value. `var` is usually not used as much anymore.

Reply:
Thanks, I will update that.